### ISSUE TITLE
Does not account for GUI scale setting

### ISSUE THREAD

**Issue by m2pt5** [jaredlll08/Controlling#74]
I usually set my GUI scale to 3, when I go into the controls with this mod installed, all the control names hang off the left side of the screen. (Reducing my GUI scale to 2 lets me see them all, but that's a workaround, not a fix.)

**Comment by jaredlll08** [comment]
What MC version are you using?

**Comment by m2pt5** [comment]
1.16.4

It may have been a side effect of one of the other mods I was using though, as I'm currently using a different pack and the keybinds menu looks fine, (I've since deleted the old pack, so I couldn't tell you what was in it.)

**Comment by jaredlll08** [comment]
Okay yea, I haven't been able to reproduce, and obviously no one else has reported it, so I don't know what it could be.

Also sorry for taking a while to reply, I never got sent a notification for this issue

**Comment by frank89722** [comment]
This happened with crash utilities installed, I also report to their Github.

**Comment by Darkere** [comment]
This is probably due to my long ass control descriptions.
I'll make those shorter, but you should probably account for that in some way.

**Comment by jaredlll08** [comment]
@Darkere link to the mod in question?

**Comment by Darkere** [comment]
https://www.curseforge.com/minecraft/mc-mods/crash-utilities

**Comment by jaredlll08** [comment]
This appears to be an issue with Vanilla, not Controlling (Should be able to reproduce with a simple resource pack changing the language).
![image](https://user-images.githubusercontent.com/5553605/106672751-41123700-65b9-11eb-9592-234d932903c8.png)

I did experiment with a few things to get it to work, such as right aligning text (and ignoring mods that have too long text).
https://i.blamejared.com/M2pnn.webm

But that just looks super weird and doesn't solve the root issue (which is that long descriptions can't be shown fully)

I also experimented with selectively right aligning text, but that just looks horrible and all over the place.
<img width="1920" alt="APHVN" src="https://user-images.githubusercontent.com/5553605/106674929-c77c4800-65bc-11eb-8b07-939550f18355.png">



At the end of the day, I can't solve an issue that vanilla itself suffers from, in any GUI (closest I ever got was with this meme https://twitter.com/jaredlll08/status/1249151871597543425, but that used gl hacks and wasn't easily scalable)
